velox/dwio/parquet/reader/PageReader.cpp

```diff
@@ -706,6 +714,31 @@ void PageReader::makeDecoder() {
               "DELTA_BINARY_PACKED decoder only supports INT32 and INT64");
       }
       break;
+    case Encoding::RLE:
+      switch (parquetType) {
+        case thrift::Type::BOOLEAN:
+          VELOX_CHECK_GE(
+              encodedDataSize_,
+              kParquetMagicNumberSize_,
+              "Received invalid length : {} (corrupt data page?)",
+              encodedDataSize_);
+          VELOX_CHECK_LE(
+              folly::Endian::little(SafeLoadAs<uint32_t>(
```

Comment:
You can use a variable and simplify like before.
auto pageDataSize = folly::Endian::little(SafeLoadAs<uint32_t>(reinterpret_cast<const uint8_t*>(pageData_));